package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cowprotocol/cow-sdk",
-  "version": "0.0.15",
+  "version": "1.0.0-RC.0",
```

Comment:
WARNING! Major version bump!

Reply:
are we going to bump the version on the same PR? @matextrem's [PR](https://github.com/cowprotocol/cow-sdk/pull/49) is also a breaking change

Reply:
I'm doing it here to be sure we don't forget this is a major version bump.
Then, if this is merged first Mati doesn't have to bump it on his